frontend/src/views/Collection/utils.tsx

```diff
@@ -119,7 +119,7 @@ export function getIsPublishable(datasets: Array<Dataset>): boolean {
 
       return (
         numOfDeployments === 1 &&
-        numOfAssets === Object.keys(DATASET_ASSET_FORMAT).length
+        numOfAssets >= Object.keys(DATASET_ASSET_FORMAT).length
```

Comment:
numOfAssets will be > data asset formats until the backend is updated

Reply:
Should we wait to merge BE's change into this PR? Thanks!

Reply:
Thinking about this a little more, maybe it's worth explicitly checking for expected formats? Feels like the total count is not reliable anymore 🤔

Reply:
@tihuan What makes you think that?

Reply:
I'm about to start working on backend changes, and I can swap back to `===` with them. Curious on why the number of assets isn't reliable outside of that case

Reply:
I was thinking about existing datasets having loom formats, so unless we clean up the database, `numOfAssets === Object.keys(DATASET_ASSET_FORMAT).length` wouldn't work in revision, since 4 > required 3

And `numOfAssets >= Object.keys(DATASET_ASSET_FORMAT).length` could cause false positive by enabling the publish button sooner, if `loom` was counted towards the threshold before the final required format becomes available. E.g.,
1. Upload dataset
2. cxg, h5ad, loom become available
3. publish button enabled <-- bug, since seurat format is not available yet
4. Thus needing to wait for BE change to be done

Something like that!

Reply:
What sort of workaround do you propose in the time between this PR and the second half of the ticket?

`numOfAssets > ASSET_KEYS.length` ?

Reply:
Once the backed half is in, we will no longer be serving the loom assets as part of the API response

Reply:
ahh I see, so loom won't show up in the API response 💡 That simplifies the check!

If that's the case, then we could use this PR as feature branch and merge the BE change in here when it's ready before merging everything, and keep the check as ` ===`?

Reply:
I separated the two parts of the ticket to help unblock launch if something goes off track. It seems like we're on path to start looking to partition tickets to have multiple deliverables, I thought this would be a good place to start

Reply:
Ahh I see! If we want to ship FE first without BE, I think we'll need to check for formats explicitly to avoid the edge cases discussed [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/1439#discussion_r717882472)

And then optionally change it back to `===` once BE guarantees no `loom` in response?

Reply:
Good idea!